packages/common/src/hardforks/regolith.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "regolith",
```

Comment:
In general maybe we should explicitly mark these hardforks as optimism? So we would use `optimism/regolith` as name (or something similar)?

Reply:
And maybe also in `Hardfork.Optimism.Regolith` as enum?

Reply:
Yeah, that makes likely sense, we can give these kind of things a bit more thinking now that we have more time on this.

Reply:
(the namespaced version might already be a good choice, good idea)